.release-please-manifest.json

```diff
@@ -10,6 +10,7 @@
   "terraform/modules/happy-route53": "1.2.1",
   "terraform/modules/happy-service-ecs": "1.3.0",
   "terraform/modules/happy-service-eks": "1.5.0",
+  "terraform/modules/happy-ingress-eks": "1.0.0",
```

Comment:
This will cause initial release to be `v1.1.0`. Omit this and let release-please autopopulate it if you want the initial release to be `v1.0.0`